README.md

```diff
@@ -1,4 +1,4 @@
-<img align="right" width="200" src="http://static.nfl.com/static/content/public/static/img/logos/react-helmet.jpg" />
+<img align="right" width="200" src="https://static.nfl.com/static/content/public/static/img/logos/react-helmet.jpg" />
```

Comment:
Seems those URLs are not working as well when opening with browser. Any other idea how to fix?